chapters/develop-organize/drills/reading/README.md

```diff
@@ -38,14 +38,59 @@ There can be essays / open-ended questions that require a personal take on a top
+1. It frees up the teachers' time to invest in the more stimulating aspects of education, such as improving their content, engaging in discussions, answering students' questions and many more.
 
-TODO: mention roles: learn / practice, self-assess, actual evaluation
+1. It provides instantaneous feedback to the students.
+The shorter feedback loop makes students more engaged in learning as they don't have to "disconnect" from a drill following its completion while waiting for human feedback.
+This promotes incremental progress regarding the drill's subject.
 
-Sample question types: [`Blackboard`](https://help.blackboard.com/Learn/Instructor/Ultra/Tests_Pools_Surveys/Question_Types), [Moodle](https://docs.moodle.org/402/en/Question_types)
+1. It aids students when studying independently.
+Connected to the previous item, the short feedback loop may introduce a gamification aspect to drills whereby students feel rewarded (sometimes using [educational cryptocurrencies](../../../infrastructure/digital-rewards/reading/README.md)) for a good portion of their individual activities.
 
-link different types of drills as `.md` files in the current directory: [questions](questions.md) and [tasks](tasks.md)
+### Subcategories
+
+While we place great importance on automated grading, we are aware that some types of drills are ill-suited for this approach.
+Therefore, we split drills into 2 categories based on the potential to automate their grading:
+
+1. [Questions](./questions.md) are drills that ask students / trainees to provide an **answer** and not a full solution.
+These answers must be predictable to the extent that this allows verifying them to be automated.
+Examples of such answers multiple-choice questions, predefined words, numbers, etc.
+Questions are the best candidates for automatic grading due to the predictability of their answers.
+
+1. [Tasks](./tasks.md) on the other hand require students / trainees to provide a **solution**, such as a proof, the steps how they reached the answer or an essay.
+While difficult to automate with regards to grading, tasks are a more profound type of drill that specifically targets the students' reasoning.
+This is why grading them should be done by humans (most of the time).
+
+The descriptions of the two types of drills outlined above are not exhaustive.
+There are in fact countless types of drills that may be used.
+For a more in-depth look, you may consult the various quiz deployment platforms in existence.
+In our opinion, two strong starting points are [Blackboard](https://help.blackboard.com/Learn/Instructor/Ultra/Tests_Pools_Surveys/Question_Types) and [Moodle](https://docs.moodle.org/402/en/Question_types)
+
+## Storage
+
+We propose that each drill be stored in a separate folder together with all supporting files, such as images, solutions, code, etc.
+This allows great flexibility when importing, exporting or extracting drills, as all files related to a particular one are grouped.
```

Comment:
Not sure if this is what you meant by flexibility, @razvand. What did you have in mind specifically?

Reply:
It's good imo